sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
```

Comment:
@pepol yes, that makes sense. I'll try to figure out how to do that while preserving backwards compatibility.

Out of blue I'd say that it would require a new decorator, something like `@subcmd` with an optional name argument. That way we could have an ordinary `@cmd` above the definition of `del_stickynote` and `@subcmd('delete')` too.